.github/workflows/wheels-rocm.yml

```diff
@@ -0,0 +1,42 @@
+name: PyTorch Triton ROCm Wheels
+on:
+  pull_request:
```

Comment:
Regarding trigger, PR is a good option. But recommend for a daily build since we have a date appended in whl version.